src/day-1-toy/slice.py

```diff
@@ -1,26 +1,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1])
 
 # Output the second-to-last element: 9
-print()
+print(a[len(a)-2])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[slice(3,6)])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print(a[slice(2,4)])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
-print()
+print(a[slice(1, 6)])
 
 # Output every element except the last one: [2, 4, 1, 7, 9]
-print()
+print(a[slice(0,5)])
 
 # For string s...
 
 s = "Hello, world!"
 
 # Output just the 8th-12th characters: "world"
-print()
\ No newline at end of file
+print(s[slice(7,12)])
```

Comment:
Another syntax you'll see commonly is `:`. For example:
```
a[slice(1,6)] === a[1:6]
```
Furthermore, if you're going to display up the end of the list, you can omit the second argument in this syntax:
```
a[1:] # Will output all elements up to the end of the list
```